tests/performance_tests/performance_test_basic.py

```diff
@@ -98,16 +104,19 @@ def cleanupOldClusters(self):
         self.cluster.killall(allInstances=self.testHelperConfig.killAll)
         self.cluster.cleanup()
 
-    def testDirsCleanup(self, saveJsonReport: bool=False):
+    def testDirsCleanup(self, delReport: bool=False):
         try:
             def removeArtifacts(path):
                 print(f"Checking if test artifacts dir exists: {path}")
                 if os.path.isdir(f"{path}"):
                     print(f"Cleaning up test artifacts dir and all contents of: {path}")
                     shutil.rmtree(f"{path}")
 
-            if saveJsonReport:
+            if not delReport:
```

Comment:
The logic of this reads inverted.  "if don't delete, do a bunch of removes" seems backwards.

Reply:
Addressed: [abc5c41](https://github.com/AntelopeIO/leap/pull/393/commits/abc5c4150d3fc732a8d6c51bfe555911d89d270b)